### ISSUE TITLE
Allow user to like only once in Q&A section

### ISSUE THREAD

**Issue by kashish-51** [HITK-TECH-Community/Community-Website#961]
In Q&A section a user can like multiple time and there is no restrictions but this should not be the case. One user should be allowed to like only once.
So, please assign this to me under GSSOC so that I can fix this.

**Comment by Kajol-Kumari** [comment]
@kashish-51 this was supposed to be fixed using the cookies in https://github.com/HITK-TECH-Community/Community-Website/pull/942 pr, i would be more than happy if u can debug and fix the issue

**Comment by kashish-51** [comment]
Thank you for assigning me

**Comment by aryansharma220** [comment]
@Kajol-Kumari I would also like to work on this issue, if needed please assign it to me also.

**Comment by Kajol-Kumari** [comment]
@kashish-51 can u please update on the progress of this issue?

**Comment by Vaishali-ajmera** [comment]
@Kajol-Kumari Please assign this task to me. I will complete it in less amount of time.

**Comment by Vaishali-ajmera** [comment]
There is some issue in the Q&A page. Page only is not showing.

**Comment by Kajol-Kumari** [comment]
> There is some issue in the Q&A page. Page only is not showing.

As its a free deployment so the backend pod takes around 50 seconds to spin up when used after a long time, wait for that time, it will come up

**Comment by Kajol-Kumari** [comment]
duplicate of https://github.com/HITK-TECH-Community/Community-Website/issues/1004
